sky/skylet/job_lib.py

```diff
@@ -22,6 +25,62 @@
 
 _JOB_STATUS_LOCK = '~/.sky/locks/.job_{}.lock'
 
+_DB_PATH = os.path.expanduser('~/.sky/jobs.db')
+os.makedirs(pathlib.Path(_DB_PATH).parents[0], exist_ok=True)
+CONN = sqlite3.connect(_DB_PATH)
+CURSOR = CONN.cursor()
+# CREATE TABLE
+try:
+    CURSOR.execute('SELECT * FROM pending_jobs LIMIT 0')
+except sqlite3.OperationalError:
+    # Tables do not exist, create them.
+    CURSOR.execute(''' CREATE TABLE pending_jobs(
```

Comment:
Should we create another `pending_jobs` table? I think we can add the `run_string` and `submit` columns to our original `jobs` table, so that we can get the job status and submit status together with
```
_CURSOR.execute('SELECT submit FROM jobs WHERE status='PENDING' ORDER_BY job_id LIMIT 1')
if not submit:
    # submit the job
else:
    return
```